### ISSUE TITLE
Item use tracking

### ISSUE THREAD

**Issue by HardwareAccelerated** [prism/Prism#75]
I'm trying to use this on a whitelisted modded server but it can't track throwables, it would be really cool if tracking item uses was an option because i want to keep no items banned but prism can't track throwable explosives, it just logs the blocks they break as broken by the "environment"

**Comment by Eufranio** [comment]
Agree.

**Comment by MrDj200** [comment]
I Agree very much. Our spawn just got griefed with Florbs. And I have no way of tracking that. Is that correct?
